src/bicep/mlz.json

```diff
@@ -18,6 +18,10 @@
       "maxLength": 24,
       "minLength": 3
     },
+"deploymentType": {
+      "type": "string",
+      "defaultValue": "MissionLandingZoneARM"
+    },
```

Comment:
Was this updated manually? Kudos, but that can prove error prone in the future should there be multiple updates to source. We should be able to generate this `src/bicep/mlz.json` file by executing `az bicep build -f src/bicep/mlz.bicep`.

Reply:
You bring up a very good point here, I should add the mlz.json to the .gitignore as this should be generated using the `bicep build `command automatically - or we need a different strategy for version control of this artifact.

Reply:
I'm not sure if ignoring the file is the right answer. We'll still need to make sure it's up to date since it's a dependency for a Portal deployment. We do have planned automation that will check if the file needs to be updated and build it for any PR submitter, however: https://github.com/Azure/missionlz/issues/426

Reply:
In the meantime, when you have a working implementation you're satisfied with, please update this file by running `az bicep build` on `mlz.bicep`.

Reply:
Confirmed, performed a bicep build on the second iteration: https://github.com/Azure/missionlz/pull/432/commits/dedb4ea579476ae3af0f45ad4dd5fd474046572e